.github/actions/ppml-spark-k8s-simplequery-on-gramine-action/action.yml

```diff
@@ -0,0 +1,70 @@
+name: 'PPML-Spark-K8S-SimpleQuery-On-Gramine'
```

Comment:
Could you put this action into .github/actions/ppml/ folder? 
`.github/actions/ppml/ppml-spark-local-simplequery-tests-on-gramine-action/action.yml`